credit-fraud-dealing-with-imbalanced-datasets.ipynb

```diff
@@ -379,10 +379,6 @@
     "\n",
```

Comment:
<p>Not sure about these changes here! Could you add these imports back please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HomelessSandwich/data-science-testing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>